contract/contracts/Library/Verify.sol

```diff
@@ -25,19 +25,24 @@ library Verify {
     uint constant strUint64Length = 8;
     uint constant enumLength = 4;
 
-    struct TypedSignature {
+    struct Signatures {
```

Comment:
Why rename it? Wouldn't it be better if it has the same name as in simperby core? If the `Typed` part is the problem, maybe rename it to `Signature` since it only has one signature.

Reply:
roll back to TypedSignature.